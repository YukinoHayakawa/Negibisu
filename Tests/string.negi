single line string

multi\
line\
 string

string {{ with a comment }}inside

\\\\ two slashes

meaningless \e\s\c\a\p\e\s
