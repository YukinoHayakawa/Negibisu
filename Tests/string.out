Token Stream
============

Line 1, Col 1: [STRING_LITERAL] single line string
Line 1, Col 19: [NEWLINE] 
Line 2, Col 1: [NEWLINE] 
Line 3, Col 1: [STRING_LITERAL] multiline string
Line 5, Col 8: [NEWLINE] 
Line 6, Col 1: [NEWLINE] 
Line 7, Col 1: [STRING_LITERAL] string inside
Line 7, Col 34: [NEWLINE] 
Line 8, Col 1: [NEWLINE] 
Line 9, Col 1: [STRING_LITERAL] \\ two slashes
Line 9, Col 17: [NEWLINE] 
Line 10, Col 1: [NEWLINE] 
Line 11, Col 1: [STRING_LITERAL] meaningless escapes
Line 11, Col 27: [NEWLINE] 

AST
===

Error at Line 1, Col 1: Expected a #.
Syntax error.
